source/release-notes.txt

```diff
@@ -10,6 +10,16 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.42.x
```

Comment:
**Note to Reviewer**

This is placeholder for the actual release version, which is TBD.

Reply:
It is more likely 1.43.x